department-of-veterans-affairs/caseflow
# Making folks without sattyid but acting judge status, return no role

See conversation: https://dsva.slack.com/archives/C6E41RE92/p1541790541077800

### Description
Users with a nil sattyid, but svlj set to "A" are treated as judges. This means they can see legacy judge tasks. However, there are lots of admin users at the board with svlj set to "A" that shouldn't see legacy judge tasks. This removes their judge role.

### Acceptance Criteria
- [ ] Users without SATTYID's with SVLJ set to "A" should not see LegacyJudgeTasks

### Testing Plan
1. Update BVAAABSHIRE's staff record
   1. SVLJ => "A"
   1. SATTYID => nil
1. Ensure when looking at a legacy case, there are no task actions.
